File: JavaScript/client_byToken.js
const ttio_udp = require('./lib/ttio_udp');

// by token

var thingToken = 'thingtokenexample';
var messageStr = 'ABCD1234';
//var sha1 = '';

ttio_udp.sendUDPMessageTTIOv1(thingToken,messageStr)
//ttio_udp.sendUDPMessageTTIOv1(thingToken,messageStr, sha1)

File: Python/client_byToken.py
import ttio_udp

# by token

thingToken = 'QnxQgWCRMMirS4kmW_6EVW7nULJ4ssnZvPU_PQ8HMNE'
messageStr = 'ABCD1234'
# var sha1 = ''

ttio_udp.sendUDPMessageTTIOv1(thingToken,messageStr)
# ttio_udp.sendUDPMessageTTIOv1(thingToken,messageStr, sha1)
